### DISCUSSION TITLE
Create Tenant via Mutation

### DISCUSSION THREAD

**Discussion by DTrafford** [django-tenants/django-tenants#550] [Q&A]
I would like to automate the deployment of new tenants via a call from another application I have.

Currently if I try to send a mutation I get:

`Can't create tenant outside the public schema. Current schema is local.`

Could you advise on how I could set this up to be possible?

**Comment by itzomen** [comment]
All tenants are saved on the public schema so if you want to make a tenant mutation make sure to call the url of the public schema since the tenants are saved there
